gemfiles/activesupport_6.0.gemfile

```diff
@@ -0,0 +1,12 @@
+# This file was generated by Appraisal
```

Comment:
not sure if we want to call this file activesupport, or if we want to just call it `gemfile`

I guess both work but if we change other gems, it may get confusing

Reply:
If these were new gems, yes! But, these are old ones, so YAGNI.  We can rename it if we add other differences we want to test with appraisal.

Reply:
I agree with @kbrock here.  The filename is confusing as is.

Reply:
What do you suggest?  ActiveSupport is the only difference for each appraisal.  If you didn't know this depends on rails or activesupport, `gemfile_6.0.gemfile` would be confusing.

It works with virtual attributes because it's called `activerecord-virtual_attributes` so it's obvious.

Reply:
Maybe `rails_6.0-gemfile` and `rails_6.1-gemfile` would be appropriate?  I thought AS would be more explicit that it's a much smaller subset of rails that's in play here.

Reply:
In activerecord-virtual_attributes they do gemfile_6.0.gemspec, but I agree with you that that also doesn't make sense. `rails_6.0.gemfile` seems better.

Reply:
Actually, I take it back...I had it in my mind that the version of _rails_ was important, but in this case it's literally activesupport and not all of rails.  rails_6.0.gemfile would be confusing.

I'm good with keeping it as is with `activesupport_6.0.gemfile `

Reply:
I like this. then if we had to update json or csv gem that works with the rails version, then it would not feel too out of place

Reply:
that argument would still make sense with the activesupport naming (i.e. "if we had to update json or csv gem that works with the activesupport version, then it would not feel too out of place")